src/filext_guesser.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EP_FILEXTGUESSER_H
+#define EP_FILEXTGUESSER_H
+
+#include <string>
+#include <unordered_map>
+
+class Meta;
+namespace FileFinder {
+	class DirectoryTree;
+}
+
+/**
+ * FilextGuesser contains helper methods for guessing the extensions used on non-standard RPG Projects.
+ */
+namespace FilextGuesser {
```

Comment:
I think there is a typo here. Should be FileExtGuesser and the file fileext_guesser.

Reply:
Hmm, I was thinking "filext", but I now think that might be trademarked by the associated ".com" address. I'll change this after work.